include/libkernelflinger/vars.h

```diff
@@ -125,9 +125,9 @@ BOOLEAN get_slot_fallback(void);
 EFI_STATUS set_slot_fallback(BOOLEAN enabled);
 
 enum device_state {
-	UNKNOWN_STATE = -1,
 	LOCKED = 0,
-	UNLOCKED = 1
+	UNLOCKED = 1,
+	UNKNOWN_STATE = 2
```

Comment:
looks like if FASTBOOT_FOR_NON_ANDROID is defined, it will cause the fastboot operation failure.
because UNKNOWN_STATE now is greater than LOCKED & UNLOCKED. 
see https://github.com/intel/kernelflinger/blob/master/libfastboot/fastboot.c#L1233